src/graphql/models/Article.js

```diff
@@ -580,6 +581,25 @@ const Article = new GraphQLObjectType({
           },
         }),
     },
+    contributors: {
+      type: new GraphQLList(Contributor),
```

Comment:
nit: can be list of non-nullable contributor.
Maybe we can make the list non-nullable as well, as long as we resolve the field to an empty array when there is no contributors.